AgentQnA/tests/1_build_images.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+WORKPATH=$(dirname "$PWD")
+export WORKDIR=$WORKPATH/../../
+echo "WORKDIR=${WORKDIR}"
+export ip_address=$(hostname -I | awk '{print $1}')
+
+
+function get_genai_comps() {
+    cd $WORKDIR
+    if [ ! -d "GenAIComps" ] ; then
+        git clone https://github.com/opea-project/GenAIComps.git
+    fi
+}
+
+function build_docker_images_for_retrieval_tool(){
```

Comment:
@minmin-intel , since 'https://github.com/opea-project/GenAIExamples/tree/main/DocIndexRetriever' example is merged. I think we might not need to build docker from scratch?